rpc/internal/perf_service.go

```diff
@@ -106,6 +106,19 @@ func (srv *perfService) AttachRollups(ctx context.Context, rollupData *RollupDat
 		return nil, newRPCError(codes.InvalidArgument, errors.Wrapf(err, "problem attaching rollup data for perf result '%s'", record.ID))
 	}
 
+	id := model.PerformanceResultSeriesId{
```

Comment:
two things: 

1. A nit. I'd prefer to have the d in ID capitalized. 
2. is there a way to have a method to produce this kind of object from the record/info type? I've seen this pattern used a lot, and it seems easy to get it wrong....

Reply:
Capitalized ID and added a `ToPerformanceResultSeriesId` to `PerformanceResultInfo`.